Auth/Auth.pm

```diff
@@ -1465,6 +1504,43 @@ sub is_cosign_active {
     return Utils::is_cosign_active();
 }
 
+# ---------------------------------------------------------------------
+
+=item Auth::Auth::Entitlement
+
+Data clas for handing the $ENV{entitlements} string.
+
+=cut
+
+# ---------------------------------------------------------------------
+package Auth::Auth::Entitlement;
+
+sub new {
+    my $class = shift;
+    my $entitlement = shift;
+
+    my $self = {};
+    $$self{entitlement_map} = {};
+    bless $self, $class;
+
+    if ( $entitlement ) {
+        $$self{entitlement_map} = map { $_ => 1 } split(/\s*;\s*/, $entitlement);
```

Comment:
I think this ends up evaluating `map` in scalar context; try:

```perl
$$self{entitlement_map} = { map { $_ => 1 } split(/\s*;\s*/, $entitlement) };
```

Reply:
🤦